modules/c/nrt/include/nrt/Utils.h

```diff
@@ -173,5 +173,24 @@ NRTPROT(void) nrt_Utils_decimalLonToGeoCharArray(double decimal, char *buffer8);
  */
 NRTAPI(char) nrt_Utils_cornersTypeAsCoordRep(nrt_CornersType type);
 
+
+/*!
+ * Helper function to actually perform a byte-swap.
+ *
+ * \param value Pointer to value being swapped
+ * \param indexOne Index of first byte to be swapped
+ * \param indexTwo Index of second byte to be swapped
+ */
+
+NRTPROT(void) nrt_Utils_swap(nrt_Uint8 *value, size_t indexOne, size_t indexTwo);
```

Comment:
Here and in the function below, the whitespace should move around the asterisk... should be `nrt_Uint8* value` instead